forgerock-auth/src/main/java/org/forgerock/android/auth/SerializableCookie.java

```diff
@@ -0,0 +1,67 @@
+/*
```

Comment:
Do you think it would fit on 'core' module? Just wondering if it could be reused by other modules in the future

Reply:
For now, I would keep in in the auth module, those classes are internal we can easily move it to core when necessary.